### PR TITLE
Improve (dev) usability of website

### PR DESCRIPTION
**Pull request by Schmavery**
- Add README.md to website explaining how to build and write a blog post
- Added a script that watches for changes and rebuilds and hotreloads the dev server.
 - Uses `nodemon` to rebuild the website on changes
 - Uses `reload` to automatically reload the browser on site rerender.
- Move posts and jobs from src/website to frontend/website and remove the symlinks. There are no more symlinks needed. A later change can maybe remove src/website if we're sure we don't need any of it for the testnet widget.

Also:
- Some tweaks to the wallet README.md (mostly just using `yarn query-fake`)

When we merge this it will probably require a little rebase by anyone currently writing a blog post.

### CODE REVIEW

File: frontend/wallet/README.md [hunk 1]
```diff
@@ -14,26 +14,30 @@ The Coda Wallet is written in Reason (https://reasonml.github.io/), and built wi
...
+
+Download and build the app:
 
 1. Clone the repo via SSH: `git clone git@github.com:CodaProtocol/coda.git`
 2. Navigate into coda/frontend/wallet
 3. Update submodules: `git submodule update --init`
 4. `yarn install` to install dependencies
-5. In a separate shell, start the fake GraphQL server: `yarn run fake`
-6. `yarn run query` to save results from introspection query in graphql_schema.json
+6. `yarn run query-fake` to generate graphql_schema.json
```
Comment (o1pranay) [inline frontend/wallet/README.md]: 👍 less steps, the better

File: frontend/wallet/README.md [hunk 2]
```diff
@@ -14,26 +14,30 @@ The Coda Wallet is written in Reason (https://reasonml.github.io/), and built wi
...
+1. Install watchman globally: `brew install watchman`
+2. Install git lfs: `brew install git-lfs`
+3. Run `git lfs install` to update hooks
+
+Download and build the app:
 
 1. Clone the repo via SSH: `git clone git@github.com:CodaProtocol/coda.git`
 2. Navigate into coda/frontend/wallet
 3. Update submodules: `git submodule update --init`
 4. `yarn install` to install dependencies
```
Comment (o1pranay) [inline frontend/wallet/README.md]: nit: maybe we can add (alternatively `npm install`) here as well

will there be any issue with packages if folks try to use npm vs yarn?

Comment (Schmavery) [reply frontend/wallet/README.md]: currently there are some issues involving npm + graphql-faker

File: frontend/website/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+### How to build the website
+
+First get the code:
+1. Clone the repo via SSH: `git clone git@github.com:CodaProtocol/coda.git`
+2. Navigate into `coda/frontend/wallet`
```
Comment (enolan) [inline frontend/website/README.md]: s/wallet/website

File: frontend/website/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+### How to build the website
+
+First get the code:
```
Comment (o1pranay) [inline frontend/website/README.md]: Just tried this - and failed because don't have pandoc

Need to add step for `brew install pandoc`

Comment (Schmavery) [reply frontend/website/README.md]: that also makes me realize I have to update the CI to handle the new build script, thanks

**Pull request merged**